uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CheatSheetsTest.java

```diff
@@ -0,0 +1,197 @@
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardFirstPage;
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardRuntimePage;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author fpospisi, djelinek
+ */
+@RunWith(RedDeerSuite.class)
+public class CheatSheetsTest {
+
+	public static final String TITLE = "Create a Fuse Integration Project";
+
+	public static final String INTRODUCTION = "This cheat sheet walks you through the steps "
+			+ "to create a new Fuse Integration project based on Spring Boot.";
+
+	public static final String PERSPECTIVE = "Select Window > Open Perspective > Fuse Integration "
+			+ "in the menu bar at the top of the workbench. This step changes the perspective "
+			+ "to set up the Eclipse workbench for development of your Apache Camel/Red Hat Fuse " + "routes.";
+
+	public static String WIZARD = "To create your new project you now need to "
+			+ "open the New Integration Project wizard. You can do that by selecting "
+			+ "File > New > Fuse Integration Project from the top menu bar.Once it "
+			+ "opened, you can specify the location your project should live in. On "
+			+ "the next page, you can select specify the target environment for deploying "
+			+ "your new project and choose a specific Camel version. On the last page, you "
+			+ "can select from a list of available templates. Please select the template "
+			+ "Simple log using Spring Boot - Spring DSL from the list.Click on Finish to "
+			+ "finally create your new project.";
+
+	@BeforeClass
+	public static void setWorkspace() {
+	}
+
+	@AfterClass
+	public static void cleanWorkspace() {
+		new CleanWorkspaceRequirement().fulfill();
+	}
+
+	/**
+	 * <p>
+	 * Fuse Integration Project cheat sheet test
+	 * </p>
+	 * <b>Steps</b>
+	 * <ol>
+	 * <li>Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse integration project</li>
+	 * <li>Check if texts in all sections are actual.</li>
+	 * <li>Activate available HyperLinks and wait for action.</li>
+	 * <li>Check according actions.</li>
+	 * <li>Go through New Project Wizard.</li>
+	 * </ol>
+	 */
+	@Test
+	public void testCheatSheets() {
+
+		boolean hasJava8 = hasJava8Available();
+
+		/*
+		 * Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse...
+		 */
+		new ShellMenuItem(new WorkbenchShell(), "Help", "Cheat Sheets...").select();
+		Shell shell = new DefaultShell("Cheat Sheet Selection");
+		DefaultTree tree = new DefaultTree(shell);
+		TreeItem item = tree.getItem("Red Hat Fuse");
+		item.expand();
+		item.getItems();
+		List<TreeItem> items = new DefaultTree().getItems();
+		items = item.getItems();
+		TreeItem toSelect = items.get(0);
+		toSelect.select();
+		new PushButton("OK").click();
+
+		/*
+		 * Open new Workbench View for Cheat Sheets
+		 */
+		WorkbenchView cheatsheets = new WorkbenchView("Cheat Sheets");
```

Comment:
it would be nice to have new class e.g. `CheatSheetsView` which could contains methods for sections, getText, links, each parts, etc... code will be then more simple, clearer and it will contains less text labels

Reply:
It would be really nice! But it sounds like - remake whole UI test! Don't worry, Gucci packet will be available soon. :-D